.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run check-format
```

Comment:
I tested your pull request. I am using MacOS but I couldn't execute the pre-commit hook